components/brave_wallet_ui/common/slices/endpoints/transaction.endpoints.ts

```diff
@@ -323,6 +323,59 @@ export const transactionEndpoints = ({
         })
     }),
 
+    sendCompressedNftTransfer: mutation<{ success: boolean }, SPLTransferFromParams>({
+      queryFn: async (payload, { endpoint }, extraOptions, baseQuery) => {
```

Comment:
can we adjust the payload type to only need the chainId instead of the entire network obj?

Reply:
I can do that, but perhaps it would be better to remove this frontend logic for, since the frontend is only partially implemented. I added this function primarily for testing

Reply:
I removed